WalletWasabi/Crypto/SchnorrBlindingSignature.cs

```diff
@@ -82,8 +82,8 @@ public uint256 BlindMessage(uint256 message, PubKey rpubkey, PubKey signerPubKey
 					goto retry;
 				}
 
-				var cp = _c.Add(_w.Negate(), out overflow); // this is sent to the signer (blinded message)
-				if (cp.IsZero || overflow != 0)
+				var cp = _c + _w.Negate(); // this is sent to the signer (blinded message)
```

Comment:
If you remove the Add and use + won't there be an uncaught exception in case of overflow?

Reply:
It is OK to "overflow" with Scalars it won't throw. The operators are just wrapped around.